app/src/main/java/org/wikipedia/search/RecentSearch.kt

```diff
@@ -0,0 +1,63 @@
+package org.wikipedia.search
+
+import android.os.Parcel
+import android.os.Parcelable
+import java.util.*
+
+class RecentSearch : Parcelable {
```

Comment:
I'm actually not sure there's any reason for this class to be `Parcelable`.  I think we can just remove everything from this class related to `Parcelable`, which is pretty much everything except the constructor.